guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -50,6 +61,39 @@ On {RHEL}, enable the `rhel-7-server-extras-rpms` repository:
 ----
 # {package-install-project} leapp leapp-repository
 ----
+ifdef::satellite[]
+. For Leapp to perform the upgrade in a disconnected environment, download the metadata and manually extract, as described in https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
+
+. Set up the following repositories to perform the upgrade in a disconnected environment:
+.. `/etc/yum.repos.d/rhel8.repo`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[BaseOS]
+name={RepoRHEL8BaseOS}
+baseurl=http://_server.example.com_/rhel8/BaseOS/
+[AppStream]
+name={RepoRHEL8AppStream}
+baseurl=http://_server.example.com_/rhel8/AppStream/
+----
++
+.. `/etc/yum.repos.d/{project-context}.repo:`
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[{RepoRHEL8ServerSatelliteServerProductVersion}]
+name={RepoRHEL8ServerSatelliteServerProductVersion}
+baseurl=http://_server.example.com_/sat6/Satellite/
+[{RepoRHEL8ServerSatelliteMaintenanceProductVersion}]
+name={RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+baseurl=http://_server.example.com_/sat6/Maintenance/
+----
++
+[NOTE]
+====
+Since Leapp completes part of the upgrade in a container that has no access to additional ISO mounts, the repositories cannot be served from a locally mounted ISO but must be delivered over the network from a different machine.
+====
```

Comment:
Should this, instead of a note, be a step in the procedure or a prerequisite to mount the ISO?

Reply:
@evgeni Your thoughts on Lena's comments here?

Reply:
Well, the whole point is, that it *cannot* be mounted as an ISO and must be served from an external machine.

At some point in the writing of this document, we had a pointer to https://access.redhat.com/solutions/5492401 which explains how an offline Leapp upgrade works in general (outside of the Satellite context). It seems to have gone missing now.

I don't mind how exactly this step and the note (if any) is formatted, but the takeaway for the reader needs to be:
* they need a dedicated machine serving repositories for RHEL (BaseOS and AppStream) and Satellite (Satellite and Maintenance)
* they absolutely cannot use the same machine for that (as they can with a normal disconnected upgrade)

Reply:
I moved this to the Prerequsites for Disconnected Environments and added https://access.redhat.com/solutions/5492401 as well.